### PR TITLE
add match info to game start block

### PR DESCRIPTION
**Pull request by NikhilNarayana**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ This is data that will be transferred as the game is starting. It includes all t
 | 0x221 + 0xA*i* | Connect Code | Shift JIS string | Connect codes used by the players in using Slippi Online. *i* is 0-3 depending on the character port. The `#` is full width (`0x8194`). All other characters are half width (1 byte). Max 7 1 byte characters + 2 byte `#` + null terminator | 3.9.0
 | 0x249 + 0x1D*i* | Slippi UID | string | Firebase UIDs of players if using Slippi Online. *i* is 0-3 depending on the character port. Max 28 characters + null terminator | 3.11.0
 | 0x2BD | Language Option | u8 | 0 = Japanese, 1 = English. Needed for HRC because stage is different between the languages | 3.12.0
+| 0x2BE | Match ID | string | An ID consisting of the mode and current time (i.e. `mode.unranked-2022-12-20T06:52:39.18-0`). Max 50 characters + null terminator | 3.14.0
```
Comment (cnkeats) [inline SPEC.md]: Can we be more descriptive about `current time`?

It's not obvious if it is the time the menu was selected, dolphin was launched, set ended, etc.

File: SPEC.md [hunk 2]
```diff
@@ -110,6 +110,9 @@ This is data that will be transferred as the game is starting. It includes all t
...
+| 0x2F1 | Game Index | u32 | Index of the game in the current set, starts at 1 | 3.14.0
+| 0x2F5 | Tiebreaker Index | u32 | Index of the tiebreaker for the current game, will be 0 if not a tiebreak game. game index should be considered when looking at this value | 3.14.0
```
Comment (cnkeats) [inline SPEC.md]:
```suggestion
| 0x2F5 | Tiebreaker Index | u32 | Index of the tiebreaker for the current game, will be 0 if not a tiebreak game. Game Index should be considered when looking at this value | 3.14.0
```

**Review by cnkeats — APPROVED** [review]

File: SPEC.md (hunk 1 above)
Comment (JLaferri) [reply SPEC.md]: It's the time the players were matched in the matchmaker.

Comment (NikhilNarayana) [reply SPEC.md]: i swapped in `time the match started`, but feel free to suggest better wording.

**Review by cnkeats — APPROVED** [review]

**Pull request merged**